### PR TITLE
add support of min and max time limit

### PR DESCRIPTION
**Pull request by okobsamoht**
add support of min and max time limit

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,162 +1,183 @@
...
+    onClick: PropTypes.func,
+    onChange: PropTypes.func,
+    showErrors: PropTypes.bool,
+    equalTimeError: PropTypes.string,
+    endTimeError: PropTypes.string,
+    onStartTimeClick: PropTypes.func,
+    onStartTimeChange: PropTypes.func,
+    onEndTimeClick: PropTypes.func,
+    onEndTimeChange: PropTypes.func,
+    minStartMoment: PropTypes.string.isRequired,
```
Comment (cadbox1) [inline src/index.js]: I think these should be optional, not required

**Review by cadbox1 — CHANGES_REQUESTED** [review]
hi @okobsamoht, thanks for the PR!

My friend, Pete, made this project but sadly he passed and handed it over to me so I'm glad you could find some value in it and I'm sure he would too.

Just a small comment about the required fields, which would be a breaking change for other users. if you could change that I'll merge it in. 

also, for future PRs it's preferable to maintain the current code style. It was pretty hard to review but I'm not too fussed and we could probably do with some code styling tools in the project.

were you interested in becoming a maintainer by any chance?

File: src/index.js (hunk 1 above)
Comment (okobsamoht) [reply src/index.js]: Hi Cadell! 
I'm sad to hear that bad news.
For the required fields ill change them to optional and commit.
I'll be glad to be part of the maintainers of this great project.
Thanks

**Comment by okobsamoht** [comment]
if you don't mind, i'll cancel the PR and do it again.

**Comment by cadbox1** [comment]
no worries, looking forward to it! thanks!

**Pull request closed without merging**